CHANGELOG.md

```diff
@@ -1,7 +1,28 @@
-DAMN VULNERABLE WEB APP
+DAMN VULNERABLE WEB APPLICATION
 =======================
 
-v1.8
+v1.9 (*Not Yet Released*)

```

Comment:
Note based on your last commit:  
`+Versioning change: Version numbers now follow Major.Minor (e.g. 1.8) removing the middle digit.`
Source: https://github.com/RandomStorm/DVWA/commit/414650738add48edac98da5a85a4365b0edfdc72#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR7